events.html

```diff
@@ -74,355 +61,413 @@ <h3>FTC Louisiana Qualifier 1
             <h3>FTC Louisiana Qualifier 3
             </h3>
             <p>The Dunham School</p>
-            <p class="text-sm">11111 Roy Emerson Dr, Baton Rouge, La 70810</p>
+            <p class="text-sm">11111 Roy Emerson Dr, Baton Rouge, LA 70810</p>
           </div>
         </div>
 
         <h2>Past Events</h2>
 
-        <div class="display-card w-full h-full grid grid-cols-3 gap-4">
-          <div>
-            <p>Fri 11/21/25</p>
-            <p>9:00am-12:00pm</p>
-          </div>
-          <div class="col-span-2">
-            <a href="https://www.intralox.com/" target="_blank" class="block-link">
-              <h3>Field Trip to Intralox
-              </h3>
-              <p>Intralox Hammond Location</p>
-              <p class="text-sm">20157 Intralox Dr, Hammond, La 70401</p>
-            </a>
+        <div class="grid grid-cols-1 grid-rows-auto gap-x-8 gap-y-8 content-start items-stretch">
+          <div class="display-card w-full h-full grid grid-cols-3 gap-4">
+            <div>
+              <p>Sat 12/13/25</p>
+              <p>7:00am-5:00pm</p>
+              <p class="text-xs w-full">
+                <a href="successes.html" class="no-underline">
+                  <!-- https://phosphoricons.com/?weight=%22fill%22&size=24&q=%22medal%22 -->
+                  <svg class="inline" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor"
+                    viewBox="0 0 256 256">
+                    <path
+                      d="M216,96A88,88,0,1,0,72,163.83V240a8,8,0,0,0,11.58,7.16L128,225l44.43,22.21A8.07,8.07,0,0,0,176,248a8,8,0,0,0,8-8V163.83A87.85,87.85,0,0,0,216,96ZM56,96a72,72,0,1,1,72,72A72.08,72.08,0,0,1,56,96Zm16,0a56,56,0,1,1,56,56A56.06,56.06,0,0,1,72,96Z">
+                    </path>
+                  </svg>
+                  Innovate Award
+                </a>
+              </p>
+              <p class="text-xs w-full">
+                <a href="successes.html" class="no-underline">
+                  <!-- https://phosphoricons.com/?weight=%22fill%22&size=24&q=%22trophy%22 -->
+                  <svg class="inline" xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor"
+                    viewBox="0 0 256 256">
+                    <path
+                      d="M232,64H208V48a8,8,0,0,0-8-8H56a8,8,0,0,0-8,8V64H24A16,16,0,0,0,8,80V96a40,40,0,0,0,40,40h3.65A80.13,80.13,0,0,0,120,191.61V216H96a8,8,0,0,0,0,16h64a8,8,0,0,0,0-16H136V191.58c31.94-3.23,58.44-25.64,68.08-55.58H208a40,40,0,0,0,40-40V80A16,16,0,0,0,232,64ZM48,120A24,24,0,0,1,24,96V80H48v32q0,4,.39,8ZM232,96a24,24,0,0,1-24,24h-.5a81.81,81.81,0,0,0,.5-8.9V80h24Z">
+                    </path>
+                  </svg>
+                  Event Winner!
+                </a>
+              </p>
+            </div>
```

Comment:
We should also add the "Qualified LA State" here. See lines 401-412 (last season's Hammond Qualifier).